tests/integration/test_ceph.py

```diff
@@ -0,0 +1,186 @@
+import time
+import unittest
+import subprocess
+from ims.common import config
+
+config.load()
+
+import rados
+import rbd
+from ims.common import shell
+from ims.common import constants
+from ims.einstein import ceph
+from ims.einstein.operations import BMI
+from ims.exception import file_system_exceptions
+from ims.common.log import trace
+import logging
+
+_cfg = config.get()
+
+POOL = _cfg.fs.pool
+CEPH_IMG = "BMI_TEST_CEPH"
+TEST_NOT_EXIST_IMG_NAME = "BMI_TEST_IMG_NOT_FOUND"
+CEPH_IMG_SIZE = 1024
+INDEX_OUTSIDE_THE_IMAGE = CEPH_IMG_SIZE + 1
+CEPH_SNAP_IMG = "BMI_TEST_CEPH_SNAP"
+TEST_DATA = 'bmi' * 10
+OFFSET = 0
+CEPH_CHILD_IMG = "BMI_TEST_CEPH_CHILD_IMAGE"
+
+
+class TestCreateImage(unittest.TestCase):
```

Comment:
Are you sure? It seems at https://github.com/CCI-MOC/ims/pull/101/files/ce7ec4a3fa2a9234af42a1d6fb8a92c82fac9598#diff-3c078ee34b92c8428b97dd0f842eacd2R35 we are creating object and init we have calls to connect https://github.com/CCI-MOC/ims/blob/dev/ims/einstein/ceph.py#L24

Or am missing something?